server/router/api.go

```diff
@@ -104,6 +104,7 @@ func apiRoutes(e *gin.RouterGroup) {
 					repo.POST("/pipelines/:number/decline", session.MustPush, api.PostDecline)
 
 					repo.GET("/logs/:number/:stepId", api.GetStepLogs)
+					repo.DELETE("/logs/:number/:stepId", session.MustPush, api.DeleteStepLogs)
```

Comment:
Could you add it to the `woodpecker-go` client as well.

Reply:
Could you provide example PR?

Reply:
#3458